zipkin-server/src/main/java/zipkin/server/ZipkinServerConfiguration.java

```diff
@@ -46,8 +48,14 @@ CollectorSampler traceIdSampler(@Value("${zipkin.collector.sample-rate:1.0}") fl
 
   @Bean
   @ConditionalOnMissingBean(CollectorMetrics.class)
-  CollectorMetrics metrics(CounterService counterService, GaugeService gaugeService) {
-    return new ActuateCollectorMetrics(counterService, gaugeService);
+  CollectorMetrics metrics(Optional<CounterBuffers> counterBuffers, Optional<GaugeBuffers> gaugeBuffers) {
```

Comment:
There is no need for standalone test, it is covered by existing integration E2E test: zipkin.server.ZipkinServerIntegrationTest#writeSpans_updatesMetrics